sbin/seqrepo-pull

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+SEQREPO_DIR="/biocommons/dl.biocommons.org/seqrepo"
+
+# pull the latest seqrepo version from biocommons
+latest_version=$(seqrepo list-remote-instances | tail -n 1 | xargs)
+cd "$SEQREPO_DIR"
+rsync -rtHP --no-motd stuart.biocommons.org::seqrepo/"$latest_version" .
```

Comment:
What is `stuart.biocommons.org`?

Reply:
It is the biocommons server. I wonder if there is a better way to pull this data. I'll check.